webapp/src/components/Select/index.js

```diff
@@ -24,7 +24,7 @@ const SelectComponent = ({
 
   return (
     <FormControl
-      sx={{ m: 1, width: width }}
+      sx={{ m: '8px 0px 8px 8px', width: width }}
```

Comment:
Please don´t use inline style code

Reply:
I deleted the margin but we need the width because this component is used from different views and from those components we send the width the component needs